pontoon/base/errors.py

```diff
@@ -0,0 +1,10 @@
+from raygun4py import raygunprovider
+
+from django.conf import settings
+
+def send_exception():
+    """
+    Function sends exception to selected provider. This will allow us to change
+    error-service in future. Currently we support Raygun.io.
+    """
+    raygunprovider.RaygunSender(settings.RAYGUN4PY_API_KEY).send_exception()
```

Comment:
You should be passing the exception in as an argument to this function and passing it on to RayGun. While RayGun is smart enough to find the last exception, it's not good to rely on that, especially for complicated error cases.